addons/l10n_latam_check/models/account_payment.py

```diff
@@ -194,7 +194,7 @@ def _compute_payment_method_line_fields(self):
         """ Add is_internal_transfer as a trigger to re-compute """
         return super()._compute_payment_method_line_fields()
 
-    @api.depends('l10n_latam_check_operation_ids.state')
+    @api.depends('l10n_latam_check_operation_ids.state', 'payment_method_line_id')
```

Comment:
Shouldn't it be
```suggestion
    @api.depends('l10n_latam_check_operation_ids.state', 'payment_method_line_id.code')
```
so that if the code is somehow changed in the method line, the journal gets recomputed (just to be safe) ?